launcher: add shutdown, reboot and logout to actions

thaemisch opened the pull request.

soramanew commented:
I actually didn't put these into the launcher on purpose so people don't accidentally shutdown/reboot/logout due to a typo or unexpected fuzzy match.

I'll merge the pr if you make them able to be optional (for now, just make some options in the `LauncherConfig.qml`).

thaemisch commented:
Further changes:
ActionItem: added variables `disabled` (bool) & `disabledReason` to show users the option is there, but not active and where to activate it

Actions: added function `handleDangerousAction` to catch dangerous actions and send a notification explaining where to activate the action
